clamav/1.0/alpine/Dockerfile

```diff
@@ -39,9 +39,9 @@ RUN apk update && apk upgrade \
         py3-pytest \
         # For Rust/Cargo
         cargo \
-        rust \
-    && \
-    mkdir -p "./build" && cd "./build" && \
+        rust
+
+RUN mkdir -p "./build" && cd "./build" && \
```

Comment:
There is a decent chance someone else will complain about this because cached layers increase the size of an image.  However, I see a strong benefit to it -- particularly for the debian Dockerfiles where the multiarch builds take a really long time. We just have to remember the reason to defend this position, when people complain about it later.

Reply:
I'm looking at our current images and seeing some older CVE's even though we run a build and push updates to the images every week.  I think that's because caching doesn't know when the `apk`-installed packages have changed. So it uses the old cached layers.  

I'm unsure how to solve this problem without forcing docker to rebuild without used the cached layers... which means that this change wouldn't help any and will only have the negative effect of increasing the size of the image by inserting those layers. 

Any ideas?